src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -10,14 +10,14 @@
     using Jalasoft.TeamUp.Resumes.Models;
     using Microsoft.Extensions.Configuration;
 
-    public class ResumeSQLRepository : IResumesRepository
+    public class ResumeSQLRepository : IResumeSQLRepository
     {
         private string connectionString;
 
         public ResumeSQLRepository()
         {
             IConfigurationRoot config = new ConfigurationBuilder()
-            .AddJsonFile("DBConfig.json", optional: false, reloadOnChange: true)
+            .AddJsonFile("local.settings.json", optional: false, reloadOnChange: true)
             .Build();
             this.connectionString = config["appSettings:ConnectionString"];
```

Comment:
I'm thinking if this is necessary? since the configuration variables are treated as environment variables we can do the same as we did in the project's repository to connect mongo, what do you think?
@alejandro-castellon-jalafund @rodrigo-baldiviezo-jalafund @niver-copa-jalafund

Reply:
Changing environment variables.

Reply:
yes too much easier

Reply:
It´s other way to do. I agree with both